openwisp_controller/connection/tests/test_connection_notification.py

```diff
@@ -0,0 +1,124 @@
+from django.apps.registry import apps
+from django.core import mail
+from django.urls import reverse
+from django.utils.html import strip_tags
+from openwisp_notifications.types import unregister_notification_type
+from swapper import load_model
+
+from .utils import CreateConnectionsMixin
+
+Notification = load_model('openwisp_notifications', 'Notification')
+Credentials = load_model('connection', 'Credentials')
+DeviceConnection = load_model('connection', 'DeviceConnection')
+
+
+class TestConnectionNotifications(CreateConnectionsMixin):
```

Comment:
also here, for consistency with the other test classes, let's call this `TestNotifications` (the other classes are named `TestModels` not `TestConnectionModels`.